gc/base/CollectorLanguageInterface.cpp

```diff
@@ -29,37 +29,12 @@
 struct OMR_VMThread;
 
 void
-MM_CollectorLanguageInterface::writeBarrierStore(OMR_VMThread *omrThread, omrobjectptr_t parentObject, fomrobject_t *parentSlot, omrobjectptr_t childObject)
+MM_CollectorLanguageInterface::writeBarrier(OMR_VMThread *omrThread, omrobjectptr_t parentObject, omrobjectptr_t childObject)

```

Comment:
Could this function be marked as MMINLINE (and leave it in the cpp) so that it gets inlined into the 2 callers.?